README.md

```diff
@@ -50,11 +50,12 @@ StripeExample.java
     import com.stripe.Stripe;
     import com.stripe.exception.StripeException;
     import com.stripe.model.Charge;
+    import com.stripe.net.RequestOptions.RequestOptionsBuilder;
 
     public class StripeExample {
 
         public static void main(String[] args) {
-            Stripe.apiKey = "YOUR-SECRET-KEY";
+            RequestOptionsBuilder builder = (new RequestOptionsBuilder()).setApiKey("YOUR-SECRET-KEY");
```

Comment:
seems a little funky to to store a RequestOptionsBuilder here instead of calling .build() and storing an actual `RequestOptions`